### PR TITLE
chore(@desktop/adminEvents): refactor admin events backend logic

### PR DESCRIPTION
**Pull request by mprakhov**
### What does the PR do

Apply status-go changes for community admin events

Closes: https://github.com/status-im/status-desktop/issues/11084

status-go PR: https://github.com/status-im/status-go/pull/3678
### Affected areas

Community admin actions

### CODE REVIEW

**Comment by mprakhov** [comment]
@glitchminer there is nothing to review on a status-desktop side as all work is done in status-go, so can we start testing without waiting for an approval from devs side

**Review by osmaczko — APPROVED** [review]

**Review by jrainville — APPROVED** [review]

**Review by glitchminer — APPROVED** [review]

**Pull request merged**
